lib/channel-postmessage/src/index.ts

```diff
@@ -14,6 +14,16 @@ interface BufferedEvent {
   reject: (reason?: any) => void;
 }
 
+const getQueryObject = (): Record<string, string & { refId?: string }> => {
+  return (window.document.location.search as string)
+    .substr(1)
+    .split('&')
+    .reduce((acc, s) => {
+      const [k, v] = s.split('=');
+      return Object.assign(acc, { [k]: v || true });
+    }, {});
+};
+
```

Comment:
You don't want to use `querystring` for this? It is a built-in.

Reply:
This code doesn't run in Node?

Reply:
You can use node libraries in webpack mate ;)

Reply:
I switched to `qs` since that's what we use everwhere for this purpose.

I tried to import `querystring` but it is missing typescript definitions...